modules/telco-ran-bios-tuning.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-bios-tuning_{context}"]
+= Host firmware tuning
+
+Description::
+Configure system level performance.
+See link:https://docs.openshift.com/container-platform/latest/scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.html#ztp-du-configuring-host-firmware-requirements_sno-configure-for-vdu[Configuring host firmware for low latency and high performance] for recommended settings.
++
+If Ironic inspection is enabled the firmware setting values are available from the per-cluster `BareMetalHost` CR on the hub cluster.
```

Comment:
Where and how is `ironic inspection` enabled?

Reply:
Not sure tbh! 

Something like this in `SiteConfig` CR?

```
      nodes:
        - hostName: "example-node1.example.com"
          role: "master"
          nodeLabels:
            node-role.kubernetes.io/example-label:
            custom-label/parameter1: true
            inspect.metal3.io: enabled
```

Reply:
To enable ironic inspection in siteconfig,
```
nodes:
- hostName: node1
  ironicInspect: "enabled"
```